src/app/store/technical-records/effects/technical-record-service.effects.ts

```diff
@@ -185,6 +194,32 @@ export class TechnicalRecordServiceEffects {
     { dispatch: false }
   );
 
+  updateAxles$ = createEffect(() =>

```

Comment:
the fact that these effects don't do anything but respond with success blindly, support my previous comment about data flow. If a new state is being created in the reducer and the component is subscribed to said state. there wouldn't be any reason for these effects.

Reply:
Frankly, this is why I wanted to consult you on Friday. I was not 100% sure whether the summary would receive the update when the state has been modified. Listening to a **success** action ensures the summary always gets the desired changes.

But if it is as you say and they are not needed, then I agree, we should go forward removing them.

Reply:
The summary component receives a tech record as an input. As long as the parent that provides it is subscribed to it throughout its life cycle e.g. `takeUntil: this.destroy` then every state change should be sent to the child (summary)

Reply:
Suggestion applied.